### PR TITLE
[BUILD] Fix source release build

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

Support build, run unit tests, and create binary tarball from source release.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

The verify steps:

1. create a source release tarball 
```
RELEASE_VERSION=1.4.0-SNAPSHOT build/release/create-package.sh source
```
2. untar tarball
```
cd work/release
tar -xzf kyuubi-1.4.0-SNAPSHOT-bin.tgz
cd kyuubi-1.4.0-SNAPSHOT-bin
```
3. compile and run unit test
```
build/mvn clean install
```
4. create binary release tarball
```
build/dist --tgz --spark-provided -Pkyuubi-extension-spark-3-1
```

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -22,7 +22,7 @@
 _config.yml export-ignore
 codecov.yml export-ignore
 dev/kyuubi-tpcds/ export-ignore
-docs/ export-ignore
+docs/imgs export-ignore
```
Comment (WillemJiang) [inline .gitattributes:25]: How can we use the imgs files?

Comment (pan3793) [reply .gitattributes:25]: It's part of the doc site, and should we suppose that users deploy the doc site from source release tarball?

Comment (pan3793) [reply .gitattributes]: Have a one-by-one check on all images, removed all unused images, and now I think there is no license issue on `docs/imgs`, thus removed this exclude rule.

**Comment by pan3793** [comment]
Thanks for review, merging to master/branch-1.3

**Pull request closed without merging**